php/libraries/NDB_Client.class.inc

```diff
@@ -103,6 +103,7 @@ class NDB_Client
             "Content-Security-Policy: "
             . "default-src 'self' 'unsafe-inline'; "
             . "script-src 'self' 'unsafe-inline' 'unsafe-eval'; "
+            . "frame-src https://player.vimeo.com/; "
```

Comment:
@driusan this is the line you need to sign off on. everything else is pretty straightforward.